lib/cors/validate.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+'use strict'
+
+const validUrl = require('valid-url')
+
+/* ::
+import type {CorsConfiguration} from './read.js'
+*/
+
+const HELP = ', see documentation for information on how to configure cors.'
+
+function validateCors (
+  cors /* : CorsConfiguration */
+) /* : Promise<CorsConfiguration> */ {
+  if (!cors) {
+    return Promise.reject(new Error('Must specify cors configuration' + HELP))
+  }
+  if (!cors.origins || !Array.isArray(cors.origins) || !cors.origins.length) {
+    return Promise.reject(new Error('Must specify at least a single allowable origin in cors configuration' + HELP))
```

Comment:
So, they should be a good production origin in, which we use for production deployments. And we automatically just use "*" for the local development server (if they've specified an origin for production) just to regain some symmetry.